Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -965,34 +962,37 @@ private void readMetadataAsFirstTree() {
      * @param upperNode
      *            TreeNodeStruct3 object
      */
-    private void readMetadataAsSecondTree(DocStruct inStrukturelement, TreeNodeStruct3 upperNode) {
+    private void readMetadataAsSecondTree(DocStructInterface inStrukturelement, TreeNodeStruct3 upperNode) {
         upperNode.setMainTitle(determineMetadata(inStrukturelement, "TitleDocMain"));
         upperNode.setZblNummer(determineMetadata(inStrukturelement, "ZBLIdentifier"));
         upperNode.setZblSeiten(determineMetadata(inStrukturelement, "ZBLPageNumber"));
         upperNode.setPpnDigital(determineMetadata(inStrukturelement, "IdentifierDigital"));
-        upperNode.setFirstImage(this.metaHelper.getImageNumber(inStrukturelement, MetadatenHelper.getPageNumberFirst()));
+        upperNode
+                .setFirstImage(this.metaHelper.getImageNumber(inStrukturelement, MetadatenHelper.getPageNumberFirst()));
         upperNode.setLastImage(this.metaHelper.getImageNumber(inStrukturelement, MetadatenHelper.getPageNumberLast()));
         // wenn es ein Heft ist, die Issue-Number mit anzeigen
-        if (inStrukturelement.getType().getName().equals("PeriodicalIssue")) {
+        if (inStrukturelement.getDocStructType().getName().equals("PeriodicalIssue")) {
             upperNode.setDescription(
-                    upperNode.getDescription() + " " + determineMetadata(inStrukturelement, "CurrentNo"));
+                upperNode.getDescription() + " " + determineMetadata(inStrukturelement, "CurrentNo"));
         }
 
         // wenn es ein Periodical oder PeriodicalVolume ist, dann ausklappen
-        if (inStrukturelement.getType().getName().equals("Periodical")
-                || inStrukturelement.getType().getName().equals("PeriodicalVolume")) {
+        if (inStrukturelement.getDocStructType().getName().equals("Periodical")
+                || inStrukturelement.getDocStructType().getName().equals("PeriodicalVolume")) {
             upperNode.setExpanded(true);
         }
 
-        // vom aktuellen Strukturelement alle Kinder in den Tree packen
-        List<DocStruct> children = inStrukturelement.getAllChildren();
+        /*
```

Comment:
Why are you changing single line comment to a multi-line comment?

Reply:
I have seen this happening from time to time. It must be a thing related to the code formatter, but I cannot really understand why it happens sometimes. Was fixed.

Reply:
Our code formatter (Eclipse -> Kitodo-IDE-formatting-Eclipse.xml) doesn't do this stuff. Do you use project specific code formatter?